backend/notes/views.py

```diff
@@ -35,4 +35,18 @@ def post(self, request):
 
 class CreateUserView(generics.CreateAPIView):
     queryset = User.objects.all()
-    serializer_class = UserSerializer
\ No newline at end of file
+    serializer_class = UserSerializer
+
+class LogInView(APIView):
+    def post(self, request):
+        username = request.data.get('login')
+        password = request.data.get('password')
+        userObjects = User.objects.all().values()
+        
+        correctCredentials = False
+        for user in userObjects:
+            if user['username'] == username and user['password'] == password:
+                correctCredentials = True
+        if correctCredentials: 
+            return Response({'message': 'Logged in'}, status=status.HTTP_201_CREATED)
+        return Response({'error': 'Wrong credentials'})
```

Comment:
return proper HTTP status, it will be good enough for the frontend, we won't need to respond with any message